egs2/TEMPLATE/asr1/scripts/utils/show_asr_result.sh

```diff
@@ -47,18 +47,25 @@ while IFS= read -r expdir; do
     if ls "${expdir}"/*/*/score_*/result.txt &> /dev/null; then
         echo "## $(basename ${expdir})"
         for type in wer cer ter; do
-            if ls "${expdir}"/*/*/score_${type}/result.txt &> /dev/null; then
+            
+		if ls "${expdir}"/*/*/score_${type}/result.txt &> /dev/null; then
                 cat << EOF
 ### ${type^^}
 
-|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
-|---|---|---|---|---|---|---|---|---|
+#|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+#|---|---|---|---|---|---|---|---|---|
```

Comment:
```suggestion
|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
|---|---|---|---|---|---|---|---|---|
```
Why did you comment out these lines?